src/Workspaces/CoreTest/SolutionTests/SolutionTests.cs

```diff
@@ -4527,7 +4527,7 @@ public async Task TestFrozenPartialSolution1()
 
             // Because we froze before ever even looking at anything semantics related, we should have no documents in
             // this project.
-            var frozenSolution = await project.Solution.WithFrozenPartialCompilationsAsync(CancellationToken.None);
+            var frozenSolution = project.Solution.WithFrozenPartialCompilations(CancellationToken.None);
```

Comment:
>WithFrozenPartialCompilations [](http://example.com/codeflow?start=50&length=29)

Out of curiosity, is the sync one preferred?

Reply:
yes.  the sync one is preferred.  if there is still an async one, i intend to remove it (can be done in followup).  the general principle we want to follow is that this is available and synchronous for code that is intended ot be in low latency features, where they don't want to even do any work on a threadpool if avoidable.